services/api/src/resources/environment/resolvers.js

```diff
@@ -364,8 +380,9 @@ const getEnvironmentHitsMonthByEnvironmentId = async (
               {
                 range: {
                   '@timestamp': {
-                    gte: `${interested_year_month}||/M`,
-                    lte: `${interested_year_month}||/M`,
+                    gte: `${interested_year_month_gte}||/M`,
+                    lte: `${interested_year_month_lte}||/M`,
```

Comment:
so according to https://codesandbox.io/s/condescending-boyd-vwpqn?fontsize=14

this will now be 
```
> 2019-12 
< 2020-01 
```
but `||/M` will then round down for `gte` and round up for `lte` (see https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html#range-query-date-math-rounding)

So I think in the end that query would actually cover two months, like `2019-12-01`-`2020-01-31`?

I think we would need to remove the `||/M` as we don't depend on ES doing the rounding anymore.

Reply:
Interesting. That may have accounted for the discrepancy in the hits I was seeing. My concern with dropping the `||/M` is a note from the original issue

> A successful request needs to contain a full timestamp YYYY-MM-DDTHH:MM:SS rather than just the YYYY-MM

Instead of dropping the rounding, I think we can just add `-1` to the gte and lte dates we send. 

Example below and updated PR.

https://codesandbox.io/s/condescending-boyd-vwpqn?fontsize=14

Reply:
I think I misunderstood how the rounding was working. I've updated this one more time.